x-pack/plugins/integration_assistant/server/templates/manifest/cel_manifest.yml.njk

```diff
@@ -10,7 +10,8 @@
       title: Dataset name
       description: |
         Dataset to write data to. Changing the dataset will send the data to a different index. You can't use `-` in the name of a dataset and only valid characters for [Elasticsearch index names](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html).
-      default: cel.cel
+      default: |
+        {{ package_name }}.{{ data_stream_name }}
```

Comment:
Doesn't this leave an `\n` at the end of the field value?

Reply:
It did not seem to , when I tested